Code/GraphMol/FileParsers/cdxml_parser_catch.cpp

```diff
@@ -634,6 +649,7 @@ TEST_CASE("CDXML") {
     int i = 0;
     for (auto &mol : mols) {
       INFO(i);
+      check_smiles(*mol, expected[i]);
       CHECK(MolToSmiles(*mol) == expected[i++]);
```

Comment:
I think this is redundant: the same thing is checked in `check_smiles()` (or whatever the new name is)